collective/soupstrainer/__init__.py

```diff
@@ -33,10 +33,10 @@ def __init__(self, exclusions=None, style_whitelist=None,
         self.class_blacklist = set(class_blacklist)
 
     def __call__(self, data):
-        if isinstance(data, basestring):
```

Comment:
There is [six.string_types](https://pythonhosted.org/six/#six.string_types) as an all Python versions replace meant for `basestring`.

Reply:
Done